src/main/java/com/github/susom/database/StatementAdaptor.java

```diff
@@ -182,6 +186,15 @@ public Object nullDate(Date arg) {
     return new Timestamp(arg.getTime());
   }
 
+  public Object nullDbDate(LocalDate arg) {
+    if (arg == null) {
+      return new SqlNull(Types.DATE);
+    }
+    Logger log = LoggerFactory.getLogger("StatementAdapter.class");
+    log.info("nullDbDate converting arg"+arg.toString()+ " to " + java.sql.Date.valueOf(arg));
```

Comment:
Sure you want to be logging here?

Reply:
I didn't.  I caught that and fixed it in the new commit, before reading this .